app/modules/transcribe/overlay/keypad-slide.directive.js

```diff
@@ -48,6 +47,25 @@ function keypadSlide(hotkeys, overlayConfig) {
             }
             textarea.caret(startTag.length + text.length);
         }
+
+        function preLoad() {
+            var deferred = $q.defer();
+            var imageArray = [];
+            for (var i = 0; i < $scope.abbreviations.length; i++) {
+                imageArray[i] = new Image();
+                imageArray[i].src = $scope.abbreviations[i].imgPath;
+            }
+            imageArray.forEach.onload = function () {
```

Comment:
I see what you mean. I thought what I was doing was to resolve the promise after _each_ image is loaded, no?
It seems fine on the site http://preview.zooniverse.org/folgerdemo/#/transcribe